src/Uno.UI/UI/Xaml/Controls/TextBox/SinglelineTextBoxView.iOS.cs

```diff
@@ -19,10 +20,10 @@ namespace Microsoft.UI.Xaml.Controls
 {
 	public partial class SinglelineTextBoxView : UITextField, ITextBoxView, DependencyObject, IFontScalable
 	{
-		private SinglelineTextBoxDelegate _delegate;
+		private SinglelineTextBoxDelegate _delegate = null!;
```

Comment:
```suggestion
		private SinglelineTextBoxDelegate _delegate;
```
and tag `Initialize()` with `[MemberNotNull(nameof(_delegate))]`

Reply:
Why?

Reply:
that way, if later `Initialized()` gets changed/no longer gets called, you still get warning/error from the compiler if `_delegate` is not being assigned a value
you are essentially voiding the benefit of having `#nullable enabled` with `null!`